.mise.toml

```diff
@@ -10,6 +9,7 @@ goreleaser = "latest"
 "go:golang.org/x/tools/cmd/goimports" = "latest"
 "go:mvdan.cc/sh/v3/cmd/shfmt" = "latest"
 "go:github.com/caarlos0/svu" = "latest"
+"npm:prettier" = "^3.5.1"
```

Comment:
I guess we'd need to specify that node needs to be installed outside of `mise`, right? That experience feels kinda _meh_. I think we should keep `bun`, or switch to `node`. Either option is fine with me (e.g., `node = "lts"`

Reply:
> If you're OK with the tradeoffs of (1), which is basically sync settings in both .editorconfig and .dprint.json, largely so that Stenciled projects don't have to depend on Yet Another Runtime ™️ by default, I can swap to that.

That sounds fine to me, tbh. I am very not attached to prettier and would be _more than happy_ to not use it.

Reply:
> > If you're OK with the tradeoffs of (1), which is basically sync settings in both .editorconfig and .dprint.json, largely so that Stenciled projects don't have to depend on Yet Another Runtime ™️ by default, I can swap to that.
> 
> That sounds fine to me, tbh. I am very not attached to prettier and would be _more than happy_ to not use it.

Updated! (Also updated the PR summary.)